docs/docs/security.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Securing Pomerium
+description: >-
+  This page is an index for security-related pages throughout Pomerium's documentation.
+---
+
+# Securing Pomerium
+
+Pomerium is a tool for securing your infrastructure, following the principles of [Zero Trust](/docs/background.md#zero-trust). But that doesn't mean that your stack is "secure" right out of the box. Additionally, security is a battle of give and take; more security often comes at the cost of more complexity, both for the administrator and the end-user. What layers of security you choose to apply (and how you configure them) is highly independent on each use case.
+
+While we can't tell you what tools and technologies are right for you, we've compiled a list of all the security-related documentation we have, organized to help you discover what path to take.
+
+## Background and Concepts
+
+If you're just getting started, we suggest reviewing the following pages:
+
+- [Background](/docs/background.md) - A quick primer on the failures of legacy models of "perimeter security", and an introduction to the concept of Zero Trust.
+- [Architecture](/docs/architecture.md) - Learn how Pomerium is broken down into component services. How you choose to deploy Pomerium will set the stage for the kind of security practices that apply to your stack.
+- [Mutual Authentication: A Component of Zero Trust](/docs/topics/mutual-auth.md) - Zero Trust's core principle could be said as "trust nothing without first (and continuously) verifying it". Mutual authentication is a big part of bringing that principle to bear. This page explains the concept and how it's achieved across several different layers of the network stack.
+- [Glossary](/docs/glossary.md) - Keep this page handy for when you run into new or unfamiliar terminology.
```

Comment:
Proposing moving this to the very top of this list.

Reply:
I disagree. I expect readers to scan this unordered list before clicking on anything so they will know it's there as needed. Putting it at the top implies that we expect them to need to open and read through it, where as keeping it at the bottom implies that it's available as a resource if needed.

Reply:
Makes sense to me.